src/common/low_precision_transformations/include/low_precision/rt_info/precisions_attribute.hpp

```diff
@@ -32,6 +32,7 @@ using PrecisionsAttributePtr = std::shared_ptr<PrecisionsAttribute>;
 class LP_TRANSFORMATIONS_API PrecisionsAttribute : public SharedValueAttribute<PrecisionsSharedValue> {
 public:
     static std::vector<ngraph::element::Type> defaultPrecisions;
+    static std::mutex defaultPrecisions_guard;
```

Comment:
1. please, rename: `defaultPrecisions_guard` => `defaultPrecisionsMutex`
2. make it private

Reply:
Done, thank you